x/ref/lib/flags/sitedefaults/sitedefaults.go

```diff
@@ -0,0 +1,7 @@
+package sitedefaults
```

Comment:
This file will actually have to be in a different Go module, otherwise I will not be able to replace just this package in go.mod. The simplest way (but not the only way) for this to be a different Go module is if this was in a new github repo.

From https://github.com/golang/go/wiki/Modules#modules
> Most often, a version control repository contains exactly one module defined in the repository root. (Multiple modules are supported in a single repository, but typically that would result in more work on an on-going basis than a single module per repository).

Reply:
why do you say 'mostly'? Let's try this first before we go the trouble of creating another repo which will be very annoying in terms of builds etc.

Reply:
Agreed. Let's merge this change as-is, and I can send a PR for the submodule after testing.

Reply:
Sent #87.